kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -196,6 +196,13 @@ data:
     #   tcp_port_25228 = 60                           # Port 25228 is used for telegraf to fluentd data
     #   tcp_port_25229 = 45                           # Port 25229 is used for telegraf to fluent-bit data in DaemonSet
 
+  hostlogs-settings: |-
+    [hostlogs_settings.geneva_logs_config]
+      environment = "VALUE_ENVIRONMENT"               # Supported values Test, Stage, DiagnosticsProd, FirstpartyProd, BillingProd, ExternalProd, CaMooncake, CaFairfax, CaBlackforest
+      account = "VALUE_ACCOUNT"                       # Name of the Geneva Logs account
+      namespace = "VALUE_ACCOUNT_NAMESPACE"           # Name of the Geneva Logs account namespace
+      configversion = "VALUE_CONFIG_VERSION"          # Version of the agent xml configuration.
+      #authid = "VALUE_AUTH_ID"                       # (Optional) Managed Identity to use for authentication. object_id#<guid>, client_id#<guid>, or mi_res_id#<identity resource id>
```

Comment:
If this is being commented out, can we make a note of why and when it should be added back (or can we remove the line entirely)?

Reply:
Added a note about when to use it